src/ProofOfConcepts/TAndCModal/tAndCModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { VoidFunctionComponent } from "react";
+import { Modal, ModalVariant, Button } from "@patternfly/react-core";
+import { Flex, FlexItem } from "@patternfly/react-core";
+
+export class SmallModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalOpen: false,
+    };
+    this.handleModalToggle = () => {
+      this.setState(({ isModalOpen }) => ({
+        isModalOpen: !isModalOpen,
+      }));
+    };
+  }
+
+  render() {
+    const { isModalOpen } = this.state;
+
+    return (
+      <React.Fragment>
+        <Button variant="primary" onClick={this.handleModalToggle}>
+          Show small modal
+        </Button>
+        <Modal
+          id="modalTerms"
+          variant={ModalVariant.small}
+          title="Terms and Conditions"
+          isOpen={isModalOpen}
+          onClose={this.handleModalToggle}
+          actions={[
+            <Button
+              key="confirm"
+              variant="primary"
+              data-testid="actionViewTerms"
+              onClick={this.handleModalToggle}
+            >
+              View Terms and Conditions
+            </Button>,
+            <Button
+              key="cancel"
+              variant="link"
+              data-testid="actionCancelViewTerms"
+              onClick={this.handleModalToggle}
+            >
+              Cancel
+            </Button>,
+          ]}
+        >
+          <Flex direction={{ default: "column" }}>
+            <FlexItem>
+              <div>
+                Red Hat has Terms and Conditions for its managed services.
+              </div>
+            </FlexItem>
+            <FlexItem>
+              <div class="pf-u-font-weight-bold">
```

Comment:
Similar as my previous comment, it would be good to use more semantic elements here. 

In this case though I think you could replace the `<div>` with `<p><strong>` elements. The `<p>` provides the semantics of this being a paragraph, but the `<strong>` provides the semantics about this being something we want emphasized. 

I don't think the `class="pf-u-font-weight-bold"` is needed if using the `<strong>` since it's redundant, but I don't think it hurts either. 🤷‍♀️

Reply:
I will also change it to strong